specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2021-10-01/examples/alertRuleTemplates/GetAlertRuleTemplateById.json

```diff
@@ -23,9 +23,6 @@
           "triggerThreshold": 0,
           "displayName": "Changes to Amazon VPC settings",
           "description": "This alert monitors changes to Amazon VPC (Virtual Private Cloud) settings such as new ACL entries and routes in route tables.\nMore information: https://medium.com/@GorillaStack/the-most-important-aws-cloudtrail-security-events-to-track-a5b9873f8255 \nand https://aws.amazon.com/vpc/",
-          "eventGroupingSettings": {
```

Comment:
both the eventGroupingSettings and entityMappings should be part of the template properties schema

Reply:
They are missing in the definitions. Will you please add them and correct the example in a separate PR?